### DISCUSSION TITLE
Blank columns, not sure why

### DISCUSSION THREAD

**Discussion by ypearson** [plasma-umass/scalene#828] [Q&A]
These columns seems always empty. Is this normal? I'm referring to the repeated memory columns and copy, only the time columns has something. Thanks

![image](https://github.com/plasma-umass/scalene/assets/13055532/48c52118-a687-48ea-99cd-0c255b69054c)

**Comment by emeryberger** [comment]
Are you running on Windows? Scalene does not support memory or copy profiling on Windows (for technical reasons).

**Reply by ypearson** [reply]
Nope, I'm using Ubuntu 22.04.4 LTS
```
$ scalene --version
Scalene version 1.5.41 (2024.05.03)
```

**Comment by emeryberger** [comment]
If the memory consumption is below a certain threshold (10MB), it doesn't register.

**Reply by ypearson** [reply]
Its definitely bigger than 10MB from this:

![image](https://github.com/plasma-umass/scalene/assets/13055532/38bf8e5e-81c2-4103-9aae-77b404bc726f)

**Reply by ypearson** [reply]
I see a spot below where it does appear, does this look right?
![image](https://github.com/plasma-umass/scalene/assets/13055532/59799198-54c7-491b-a6aa-73ff17480ecb)

**Reply by emeryberger** [reply]
Looks plausible.
